mi/dataset/driver/zplsc_c/zplsc_c_echogram.py

```diff
@@ -132,7 +132,7 @@ def _setup_plot(self, depth_range, max_depth):
         yticks = np.linspace(0, max_depth, self.num_yticks)
 
         # create range vector (depth in meters)
-        yticklabels = np.round(np.linspace(depth_range[-1], depth_range[0], self.num_yticks)).astype(int)
+        yticklabels = np.round(np.linspace(depth_range[0][-1], depth_range[0][0], self.num_yticks)).astype(int)
```

Comment:
hah, i initially thought this said yicklabels

Reply:
yucklables would have been better

Reply:
With my dyslexia, I thought it said lickylables ... :)